src/main/java/org/generationcp/middleware/service/impl/inventory/LotServiceImpl.java

```diff
@@ -248,6 +230,28 @@ public void closeLots(final Integer userId, final List<Integer> lotIds) {
 		this.daoFactory.getLotDao().closeLots(lotIds);
 	}
 
+	@Override
+	public void mergeLots(final Integer userId, final Integer keepLotId, final LotsSearchDto lotsSearchDto) {
+		//Search selected lots to be merged and remove the one to keep
+		List<ExtendedLotDto> extendedLotDtos = this.searchLots(lotsSearchDto, null).stream()
```

Comment:
we also use final for the variables, something we need to discuss but it is the standard we have until now